draft-bbs-signatures.md

```diff
@@ -604,11 +600,11 @@ Procedure:
 
 8. cv = HASH(PK || Abar || A' || D || C1 || C2 || pm)
```

Comment:
I'm noticing this everywhere but curious about the best approach. All points and scalars that will be used in pairing operations should be checked in verify functions for zero and infinity to avoid returning true for invalid signatures when this is the case. How should we add this method check?

Reply:
That's a good point. So that means A, B, A' and Abar then (since W is already checked for 0 and infinity)?? Maybe a security consideration or an addition to the considerations section describing that all elements passed to the pairings must be checked accordingly (similar to the hash inputs lengths)??

Reply:
That's a good start. For SpkVerify this means A', Abar, and D and scalars. For verify this means A and scalars. The public key should always be checked.

Reply:
Referenced in #126